src/renderers/webgl/WebGLAttributes.js

```diff
@@ -19,7 +19,7 @@ function WebGLAttributes( gl, capabilities ) {
 
 		let type;
 
-		if ( array instanceof Float32Array ) {
+		if ( array instanceof Float32Array && array instanceof Float64Array ) {
```

Comment:
This change breaks all float buffer attribute usage because now an array has to be of type `Float32Array` AND `Float64Array` (which is never true).

Reply:
Does the type need to be changed to gl.DOUBLE?

Reply:
`gl.DOUBLE` does not exist.

My point is your code is wrong right now and obviously untested. The line should be correct to:
```js
if ( array instanceof Float32Array || array instanceof Float64Array ) {
```